RCPTT_Tests/GroupTests/WhenAConfigIsSavedAsAComponentThenComponentGroupsAreRemoved.test

```diff
@@ -0,0 +1,35 @@
+--- RCPTT testcase ---
+Format-Version: 1.0
+Element-Name: WhenAConfigIsSavedAsAComponentThenComponentGroupsAreRemoved
+Element-Type: testcase
+Element-Version: 3.0
+External-Reference: 
+Id: _I072oExYEeehx9brc2-KyQ
+Runtime-Version: 2.1.0.201606221726
+Save-Time: 6/9/17 10:36 AM
+Testcase-Type: ecl
+
+------=_.content-0a7243a0-75d3-3d5f-9791-539de0e5b7ac
+Content-Type: text/ecl
+Entry-Name: .content
+
+let [val config_name [test_prefix "group_test_config"]] [val comp_name [test_prefix "group_test_comp"]] [val config_block "config_block" ] [val comp_block_1 "comp_block_1"] 
+	[val comp_block_2 "comp_block_2"] [val config_group "CONFIG_GROUP"] [val comp_group_1 "COMP_GROUP_1"] [val comp_group_2 "COMP_GROUP_2"]
+	[val as_component_name [test_prefix "config_as_component"]]  {
+	
+	// create_simple_config_tree_with_groups $config_name $comp_name $config_block $comp_block_1 $comp_block_2 $config_group $comp_group_1 $comp_group_2
```

Comment:
Ooops, yes. I just got rid of it during testing to avoid trying to crate duplicate trees